.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -587,6 +594,11 @@ stages:
         testResultsFiles: tracer/build_data/results/**/*.trx
       condition: succeededOrFailed()
 
+    - template: steps/run-in-docker.yml
+      parameters:
+        baseImage: $(baseImage)
+        command: "CheckBuildLogsForErrors"
```

Comment:
Continue on error required here?

Reply:
(and for the other additions in this file?)

Reply:
I specifically don't want `continueOnError` as I want to fail the build if we have unexpected errors in the log files